analysis_notebooks/Object Analysis.ipynb

```diff
@@ -290,14 +294,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "data_folder = None\n",
-    "info = pickle.load(open(\"results/{}/obj_cnt.pkl\".format(folder_name), \"rb\"))\n",
-    "sample_file = info['filepaths'][0][0]\n",
-    "if not os.path.exists(sample_file):\n",
-    "    assert data_folder is not None, \"initialize data_folder with folder path of your data\"\n",
-    "    dataset.init_folder_path(data_folder)\n",
-    "    print(\"overwriting from_path() function\")\n",
-    "    dataset.from_path = dataset.from_path_prerun"
+    "# data_folder = None\n",
```

Comment:
why is this all commented out?

Reply:
That's my bad I will uncomment, I commented because it causes issues with the CocoDatasetNoImages when I test locally. Specifically it is looking for sample_file: /n/fs/visualai-scr/Data/Coco/2014data/train2014/COCO_train2014_000000321860.jpg